src/modules/estore.module.ts

```diff
@@ -1,13 +1,21 @@
 import { Module } from '@nestjs/common';
+import { APP_INTERCEPTOR } from '@nestjs/core';
 import { AuthModule } from '@ukef/modules/auth/auth.module';
 
+import { DtfsStorageExceptionTransformInterceptor } from './dtfs-storage/dtfs-storage-exception-transform.interceptor';
 import { GraphModule } from './graph/graph.module';
 import { GraphClientModule } from './graph-client/graph-client.module';
 import { SiteModule } from './site/site.module';
+import { TemporaryFileCheckModule } from './temporary-file-check/temporary-file-check.module';
 
 @Module({
-  imports: [AuthModule, GraphClientModule, GraphModule, SiteModule],
-  providers: [],
+  imports: [AuthModule, GraphClientModule, GraphModule, SiteModule, TemporaryFileCheckModule],
+  providers: [
+    {
+      provide: APP_INTERCEPTOR,
+      useClass: DtfsStorageExceptionTransformInterceptor,
+    },
+  ],
```

Comment:
We're registering the interceptor globally here, did you consider registering the interceptor per-endpoint instead?

Reply:
I realised I should've registered it per-endpoint yesterday after I raised this PR. I've changed it